poolrpc/auctioneer.proto

```diff
@@ -441,48 +441,56 @@ message OrderMatchPrepare {
     batch.
     */
     uint32 clearing_price_rate = 2;
+}
+
+message OrderMatchPrepare {
+    /*
+    Maps the distinct lease duration markets to the orders that were matched
+    within and the discovered market clearing price.
+    */
+    map<uint32, MatchedMarket> matched_markets = 1;
 
     /*
     A list of the user's own accounts that are being spent by the matched
     orders. The list contains the differences that would be applied by the
     server when executing the orders.
     */
-    repeated AccountDiff charged_accounts = 3;
+    repeated AccountDiff charged_accounts = 2;
```

Comment:
I think we should avoid shifting the protos here. Feels like it could lead to some unexpected issues.

Reply:
Yeah, made all changes non-breaking for older clients.